core/common/src/main/java/alluxio/PropertyKey.java

```diff
@@ -11,842 +11,649 @@
-  OSS_ENDPOINT_KEY(Name.OSS_ENDPOINT_KEY, null),
-  OSS_SECRET_KEY(Name.OSS_SECRET_KEY, null),
-  S3A_ACCESS_KEY(Name.S3A_ACCESS_KEY, null),
-  S3A_SECRET_KEY(Name.S3A_SECRET_KEY, null),
-  S3N_ACCESS_KEY(Name.S3N_ACCESS_KEY, null),
-  S3N_SECRET_KEY(Name.S3N_SECRET_KEY, null),
-  SWIFT_API_KEY(Name.SWIFT_API_KEY, null),
-  SWIFT_AUTH_METHOD_KEY(Name.SWIFT_AUTH_METHOD_KEY, null),
-  SWIFT_AUTH_URL_KEY(Name.SWIFT_AUTH_URL_KEY, null),
-  SWIFT_PASSWORD_KEY(Name.SWIFT_PASSWORD_KEY, null),
-  SWIFT_SIMULATION(Name.SWIFT_SIMULATION, null),
-  SWIFT_TENANT_KEY(Name.SWIFT_TENANT_KEY, null),
-  SWIFT_USE_PUBLIC_URI_KEY(Name.SWIFT_USE_PUBLIC_URI_KEY, null),
-  SWIFT_USER_KEY(Name.SWIFT_USER_KEY, null),
-  SWIFT_REGION_KEY(Name.SWIFT_REGION_KEY, null),
+  public static final PropertyKey GCS_ACCESS_KEY =
+      create(ConstantPropertyKey.Name.GCS_ACCESS_KEY, null);
+  public static final PropertyKey GCS_SECRET_KEY =
+      create(ConstantPropertyKey.Name.GCS_SECRET_KEY, null);
+  public static final PropertyKey OSS_ACCESS_KEY =
+      create(ConstantPropertyKey.Name.OSS_ACCESS_KEY, null);
+  public static final PropertyKey OSS_ENDPOINT_KEY =
+      create(ConstantPropertyKey.Name.OSS_ENDPOINT_KEY, null);
+  public static final PropertyKey OSS_SECRET_KEY =
+      create(ConstantPropertyKey.Name.OSS_SECRET_KEY, null);
+  public static final PropertyKey S3A_ACCESS_KEY =
+      create(ConstantPropertyKey.Name.S3A_ACCESS_KEY, null);
+  public static final PropertyKey S3A_SECRET_KEY =
+      create(ConstantPropertyKey.Name.S3A_SECRET_KEY, null);
+  public static final PropertyKey S3N_ACCESS_KEY =
+      create(ConstantPropertyKey.Name.S3N_ACCESS_KEY, null);
+  public static final PropertyKey S3N_SECRET_KEY =
+      create(ConstantPropertyKey.Name.S3N_SECRET_KEY, null);
+  public static final PropertyKey SWIFT_API_KEY =
+      create(ConstantPropertyKey.Name.SWIFT_API_KEY, null);
+  public static final PropertyKey SWIFT_AUTH_METHOD_KEY =
+      create(ConstantPropertyKey.Name.SWIFT_AUTH_METHOD_KEY, null);
+  public static final PropertyKey SWIFT_AUTH_URL_KEY =
+      create(ConstantPropertyKey.Name.SWIFT_AUTH_URL_KEY, null);
+  public static final PropertyKey SWIFT_PASSWORD_KEY =
+      create(ConstantPropertyKey.Name.SWIFT_PASSWORD_KEY, null);
+  public static final PropertyKey SWIFT_SIMULATION =
+      create(ConstantPropertyKey.Name.SWIFT_SIMULATION, null);
+  public static final PropertyKey SWIFT_TENANT_KEY =
+      create(ConstantPropertyKey.Name.SWIFT_TENANT_KEY, null);
+  public static final PropertyKey SWIFT_USE_PUBLIC_URI_KEY =
+      create(ConstantPropertyKey.Name.SWIFT_USE_PUBLIC_URI_KEY, null);
+  public static final PropertyKey SWIFT_USER_KEY =
+      create(ConstantPropertyKey.Name.SWIFT_USER_KEY, null);
+  public static final PropertyKey SWIFT_REGION_KEY =
+      create(ConstantPropertyKey.Name.SWIFT_REGION_KEY, null);
+
+  //
+  // Mount table related properties
+  //
+  public static final PropertyKey MASTER_MOUNT_TABLE_ROOT_ALLUXIO =
```

Comment:
mount table is required to have the root (/) entry, and we will explicitly check it